concent_api/core/views.py

```diff
@@ -225,19 +225,6 @@ def receive(request, _message):
             decoded_message_data.sig = None
             return decoded_message_data
         return None
-    else:
```

Comment:
You're deleting the `else` branch and not adding anything in its place. What happens when you get some other message type? Is it possible? If not you should replace this line above:

``` python
if isinstance(decoded_message_data, message.RejectReportComputedTask):
```
with
``` python
assert isinstance(decoded_message_data, message.RejectReportComputedTask)
```

and a comment or a message in `assert` explaining why you think you won't get any other message type here.

Reply:
If statement changed to assert and commented.